x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiPageTemplate,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { PLUGIN_ID } from '../../../common';
+import { useKibanaServices } from '../hooks/use_kibana';
+import { CodeBox } from './code_box';
+import { javascriptDefinition } from './languages/javascript';
+import { languageDefinitions } from './languages/languages';
+import { LanguageDefinition } from './languages/types';
+import { InstallClientPanel } from './overview_panels/install_client';
+import { OverviewPanel } from './overview_panels/overview_panel';
+import './overview.scss';
+import { IngestData } from './overview_panels/ingest_data';
+
+export const ElasticsearchOverview = () => {
+  const [selectedLanguage, setSelectedLanguage] =
+    useState<LanguageDefinition>(javascriptDefinition);
+  const { http } = useKibanaServices();
+
+  return (
+    <EuiPageTemplate offset={0} grow restrictWidth>
+      <EuiPageTemplate.Section alignment="top" className="serverlessSearchHeaderSection">
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup justifyContent="flexEnd" direction="column" gutterSize="l">
+              <EuiFlexItem grow={false}>
+                <EuiTitle className="serverlessSearchHeaderTitle" size="l">
+                  <h1>
+                    {i18n.translate('xpack.serverlessSearch.header.title', {
+                      defaultMessage: 'Get started with Elasticsearch',
+                    })}
+                  </h1>
+                </EuiTitle>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiText>
+                  {i18n.translate('xpack.serverlessSearch.header.description', {
+                    defaultMessage:
+                      "Set up your programming language client, ingest some data, and you'll be ready to start searching within minutes.",
+                  })}
+                </EuiText>
+                <EuiSpacer size="xxl" />
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiImage
+              alt=""
```

Comment:
I'm assuming we'll want to add a value for this later?

Reply:
Nope, decorative images should have an empty string as alt text: https://www.w3.org/WAI/tutorials/images/decorative/